sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/sas/AccountSasPermission.java

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.storage.common;
+package com.azure.storage.common.sas;
+
+import com.azure.storage.common.SR;
```

Comment:
There is another PR that removes this class, so there will be a merge conflict here probably.